src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PersonNameTest.java

```diff
@@ -62,8 +62,9 @@ void testToEdifactWithBlankIdentificationAndBlankNamesThrowsException() {
     @Test
     void testFromString() {
         assertAll("fromString",
-            () -> assertEquals(IDENTIFICATION_ONLY_VALUE, PersonName.fromString(IDENTIFICATION_ONLY).getValue()),
-            () -> assertEquals(IDENTIFICATION_AND_NAMES_VALUE, PersonName.fromString(IDENTIFICATION_AND_NAMES).getValue()),
+            () -> assertEquals(ID_ONLY_VALUE, PersonName.fromString(ID_ONLY).getValue()),
+            () -> assertEquals(ID_AND_NAMES_VALUE,
```

Comment:
Change not needed I think

Reply:
The change was made to fix a long line elsewhere - this line being changed is just a side-effect.